src/InterNations/Sniffs/Syntax/QueryParamSniff.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace InterNations\Sniffs\Syntax;
+
+use PHP_CodeSniffer_File as CodeSnifferFile;
+use PHP_CodeSniffer_Sniff as CodeSnifferSniff;
+
+class QueryParamSniff implements CodeSnifferSniff
+{
+    private static $requiredAttributes = [
```

Comment:
Array constants are supported from PHP 5.6. And there are travis builds for 5.4 and 5.5, that will fail.